charts/application/templates/k8s-rolebinding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- range .Values.serviceAccount.rbac }}
+{{- if or (eq .kind "RoleBinding") (eq .kind "ClusterRoleBinding") }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: {{ .kind }}
+metadata:
+  name: {{ $.Release.Name }}
```

Comment:
that will not work, because multiple bindings all would have the same name.
maybe add the roleName as name suffix

Reply:
done